src/client/rsg-components/Examples/ExamplesRenderer.js

```diff
@@ -7,12 +7,17 @@ const styles = () => ({
 	root: {},
 });
 
-export function ExamplesRenderer({ classes, children }) {
-	return <article className={classes.root}>{children}</article>;
+export function ExamplesRenderer({ classes, name, children }) {
+	return (
+		<article className={classes.root} data-testid={name + '-examples'}>
```

Comment:
Let's use template literals for consistency here and in other palaces:

```suggestion
		<article className={classes.root} data-testid={`${name}-examples`}>
```